packages/react-components/src/components/chart/chartOptions/style/convertStyles.ts

```diff
@@ -52,28 +52,27 @@ export const getChartStyleSettingsFromMap =
  * @returns [StyleSettingsMap, (datastream: DataStream) => ChartStyleSettingsOptions]
  */
 export const useChartStyleSettings = (datastreams: DataStream[], chartOptions: ConvertChartOptions) => {
+  // read from store
   const highlightedDataStreams = useChartStore((state) => state.highlightedDataStreams);
-  const isDataStreamHighlighted = isDataStreamInList(highlightedDataStreams);
-
   const hiddenDataStreams = useChartStore((state) => state.hiddenDataStreams);
-  const isDataStreamHidden = isDataStreamInList(hiddenDataStreams);
 
+  // Dependencies to string
```

Comment:
Would it be worth further explaining why you're doing this with a comment?

Reply:
let me add some comments